sirepo/job_api.py

```diff
@@ -100,6 +100,12 @@ async def api_downloadDataFile(
                     req_data=req.req_data,
                     suffix=s,
                 )
+                if r.state == "canceled":
+                    # POSIT: Users can't cancel donwloadDataFile. So canceled means there was a
+                    # timeout (max_run_secs exceeded).
+                    raise sirepo.util.DownloadDataFileTimeout(
+                        f"sim_type={req.type} sid={req.id} report={req.req_data.report}"
```

Comment:
Do we want to provide this information to the user or is this logging? If it is logging, don't use an fstring.

Reply:
5083fc773